vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -348,6 +389,23 @@ void vtkF3DImguiActor::RenderDropZone()
     /* Use background draw list to prevent "ignoring" NoBringToFrontOnFocus */
     ImDrawList* draw_list = ImGui::GetBackgroundDrawList();
 
+    if (this->DropZoneLogoVisible && this->Pimpl->LogoTexture)
+    {
+      // Calculate logo position (centered)
+      ImVec2 center = viewport->GetWorkCenter();
+
+      // Choose desired size (e.g., 128x128 or scale as you want)
```

Comment:
who is you in this comment ?
Instead explain why you choose that size if that matters ?